Library/Formula/cairo.rb

```diff
@@ -22,7 +21,6 @@ class Cairo < Formula
   depends_on "libpng"
   depends_on "pixman"
   depends_on "glib"
-  depends_on :x11 => :recommended
```

Comment:
I vote no. Let's slay the X11 monster and keep it from showing its ugly head ever again in the gtk+ dependency graph.